assets/js/admin/meta-boxes-product-variation.js

```diff
@@ -447,8 +447,13 @@ jQuery( function( $ ) {
 		 * @return {Object}
 		 */
 		get_variations_fields: function( fields ) {
-			var data = $( ':input', fields ).serializeJSON();
-
+			
+			var data = {};
+			
+			$( ':input', fields ).each( function(index, elem) {
+				data[elem.name] = elem.value;
```

Comment:
It works with inputs that contain names like `name[]`?

Reply:
@claudiosanches 
Yes, it works. I've debugged the Ajax call and the values received on the server side are exactly the same as when using the serializeJSON.
So all the names with [] are received as an array of values.

And, on my case, just replacing the serializeJSON decreased the delay before the Ajax call is submitted from 30 seconds to just 1 second.

Reply:
Since the AJAX call that consumes this doesn't use a JSON data type, we don't need to worry about the structure of the fields. PHP on the server will parse any arrays accordingly, and we don't need to waste the performance.